scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -25,9 +39,15 @@ const manifestIdsToIndex = () => {
   return JSON.parse(contents).manifests
 }
 
+const determineCentury = (year) => {
+  return parseInt(year / 100, 10) + 1
+}
+
 const getSearchDataFromManifest = (manifest) => {
   const identifier = manifest.id.replace(/http[s]?:\/\/.*?\//, '').replace('/manifest', '').replace('collection/', '')
-
+  const date = Math.floor(1700 + Math.random() * 300)
+  const centuryTag = determineCentury(date) + 'thcentury'
```

Comment:
I know this is pedantic, but can we think of a better name to represent century tags? Obviously it's easier to just suffix them all the same, but 1th, 2th, and 3th... don't read too well. :) I'm not sure if using the proper English suffixes solves this or if it might present unnecessary technical burden in other areas.

Reply:
right well, I sort of consider this to be just an experiment.  Can I get tags in and does it solve the problem I need.  If it does I will include moment in the project and add proper suffixes when I finalize the date tagging system.  

Short answer is I don't think it is pedantic at all to put the proper data in.